build-images.sh

```diff
@@ -27,8 +27,8 @@ buildah add "${container}" imageroot /imageroot
 buildah add "${container}" ui/dist /ui
 # Setup the entrypoint, ask to reserve one TCP port with the label and set a rootless container
 buildah config --entrypoint=/ \
-    --label="org.nethserver.authorizations=traefik@any:routeadm" \
-    --label="org.nethserver.tcp-ports-demand=3" \
+    --label="org.nethserver.authorizations=node:fwadm traefik@any:routeadm" \
```

Comment:
:grey_question: does the module need to admin HTTP routes on ANY cluster of the node?

```suggestion
    --label="org.nethserver.authorizations=node:fwadm traefik@node:routeadm" \
```